RecoHGCal/TICL/plugins/SimTrackstersProducer.cc

```diff
@@ -95,13 +99,17 @@ void TrackstersFromSimClustersProducer::fillDescriptions(edm::ConfigurationDescr
   descriptions.addWithDefaultLabel(desc);
 }
 
-void TrackstersFromSimClustersProducer::produce(edm::Event& evt, const edm::EventSetup& es) {
-  auto result = std::make_unique<std::vector<Trackster>>();
+void SimTrackstersProducer::produce(edm::Event& evt, const edm::EventSetup& es) {
+  auto result = std::make_unique<TracksterCollection>();
   auto output_mask = std::make_unique<std::vector<float>>();
+  auto result_fromCP = std::make_unique<TracksterCollection>();
+  auto output_mask_fromCP = std::make_unique<std::vector<float>>();
+  auto cpToSc_STS = std::make_unique<std::map<uint, std::vector<uint>>>();
```

Comment:
I understand what is cpToSc (caloParticleToSimCluster), but what is STS? Perhaps being somewhat more verbose here with the variable name may help future readers of this code.